storage/series/series.go

```diff
@@ -658,6 +667,7 @@ func (s *dbSeries) Reset(
 	defer s.Unlock()
 
 	s.id = id
+	s.tags = tags
```

Comment:
Can you also set `s.tags = nil` in the series Close method?

Reply:
sure thing

Reply:
done